### PR TITLE
Arabic language support

### PR DESCRIPTION
**Pull request by afan02**
Added Arabic localization of text.

### CODE REVIEW

**Comment by bfjelds** [comment]
Awesome!  We will work with our localization team to validate the resources and then merge it.  Thanks for the contribution!

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/ar-SA/Resources.resw [hunk 1]
```diff
@@ -0,0 +1,718 @@
...
+  <data name="BluetoothAttemptingToPairFormat" xml:space="preserve">
+    <value>جاري محاولة الربط مع {0} ({1})</value>
+    <comment>Format string for message displayed when pairing attempt starts</comment>
+  </data>
+  <data name="BluetoothConfirmOnlyFormat" xml:space="preserve">
+    <value>جاري ربط {0} ({1})</value>
+    <comment>Format string for message displayed when ConfirmOnly ceremony is invoked</comment>
+  </data>
+  <data name="BluetoothConfirmPinMatchFormat" xml:space="preserve">
+    <value>هل يتطابق رقم PIN الظاهر على الجهاز مع {0} ؟<value>
```
Comment (MahmoudGSaleh) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/ar-SA/Resources.resw]: The closing tag here is missing a "/". Should be 
"<value>هل يتطابق رقم PIN الظاهر على الجهاز مع {0} ؟</value>".
Can you please add it?
Thanks!

File: IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -318,7 +318,7 @@
     <value>The first, and most basic thing a board should do is blink.</value>
   </data>
   <data name="TutorialBlinkyBody2Text" xml:space="preserve">
-    <value>As mentioned in the "Get Started" section, all the source code for this open source and available on www. windowsondevices.com (look for the IoTCoreDefaultApp).</value>
+    <value>As mentioned in the "Get Started" section, all the source code for this open source are available on www.windowsondevices.com (look for the IoTCoreDefaultApp).</value>
```
Comment (bfjelds) [inline IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/en-US/Resources.resw]: Great catch ... thank you!  The wording we intended was: "all of the source code for this is open source and is available on"

**Comment by afan02** [comment]
There. Fixed both issues commented :)

**Comment by MahmoudGSaleh** [comment]
:+1:

And, the localization to Arabic looks great. It will benefit a lot of users.
Thank you!

**Pull request merged**